awx/api/generics.py

```diff
@@ -100,6 +100,7 @@ def post(self, request, *args, **kwargs):
 
             return ret
         else:
+            logger.warn(smart_text(u"Login failed for user {}".format(self.request.POST['username'])))
```

Comment:
```python
if 'username' in self.request.POST:
    ...
```

Reply:
Thank you for your review! I have fixed and passed `make test` on my end:

```